app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentContentCardTest.kt

```diff
@@ -0,0 +1,31 @@
+package org.oppia.app.player.state
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.hasDescendant
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.home.HomeActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPosition
+
+// TODO(#205): Add test case for image parsing once PR #205 is merged.
```

Comment:
I think you can link this TODO to the issue that you created yesterday.

Reply:
Added Todo to the issue #277 .